### PR TITLE
Add config schema test for guidelines to adding config

### PR DESCRIPTION
**Pull request by j16r**
Screen Shot 2021-06-08 at 12 39 42

### CODE REVIEW

File: core/store/orm/schema_test.go [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+		item, found := items[field.Name]
+
+		//
+		// ╭──╮   ╭────────────────────────────────────╮
+		// │  │   │ It looks like you're trying to add │
+		// @  @  ╭│ configuration variable!            │
+		// ││ ││ ││                                    │
+		// ││ ││ ╯╰────────────────────────────────────╯
+		// │╰─╯│
+		// ╰───╯
```
Comment (samsondav) [inline core/store/orm/schema_test.go:161]: 😄

**Review by samsondav — APPROVED** [review]
Not all config variables should be printed, maybe we want to make a note of that?

**Review by connorwstein — APPROVED** [review]

File: core/store/orm/schema_test.go (hunk 1 above)
Comment (spooktheducks) [reply core/store/orm/schema_test.go:161]: Bahahahaha wowww

File: core/store/orm/schema_test.go [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+		// variable, please make sure to:
+		//
+		// 1. Update the changelog
+		// 2. Update the ConfigPrinter found in core/store/presenters/presenters.go
+		//    if you think this variable needs to be shown in the UI
+		// 3. Update the documentation page
+		// 4. Add your new config variable to this test
+		//
+
+		require.True(t, found, fmt.Sprintf("New test variable: '%s', see test comment for guide on steps to follow when adding a configuration variable", field.Name))
```
Comment (spooktheducks) [inline core/store/orm/schema_test.go:173]: Should put Clippy in the test failure method :grin:

**Review by spooktheducks — COMMENTED** [review]
Should put Clippy in the test failure method :grin:

File: core/store/orm/schema_test.go (hunk 2 above)
Comment (j16r) [reply core/store/orm/schema_test.go:173]: lol might do that in a follow up

**Pull request merged**